doc/htmldoc/getting_help.rst

```diff
@@ -33,9 +33,5 @@ Model information
 
    nest.synapse_models
 
-* To get details on model equations and parameters, use:
-
-.. code-block:: python
-
-   nest.help('model_name')
-
+* To get details on the model equations and parameters,
+  please check our documentation at https://nest-simulator.readthedocs.io/.
```

Comment:
```suggestion
  please check out our `online model documentation <https://nest-simulator.readthedocs.io/en/stable/models/index.html>`_.
```

Reply:
Thank you for your input. The path to the model documentation could change over time. We would then have to check the path with every release.  I avoid that with the general link. Or do you think that's not a problem?